### ISSUE TITLE
Ambiguity regarding GISTIC score

### ISSUE THREAD

**Issue by beginner984** [PoisonAlien/maftools#481]
Hi

I have plotted my GISTIC2 data by 

`gisticChromPlot(gistic = laml.gistic, markBands = "all")`

And I obtained such a plot

![Picture1](https://user-images.githubusercontent.com/43682980/76239933-4987b380-622a-11ea-847f-3404ad986bb6.png)

We are seeing some peak with negative GISTIC score (deletion)

But, when I look at my scores.gistic output file by GISTIC I am seeing no negative value in score column

I am wondering from where peaks with negative scores come when all score in gistic score file are positive?
[scores.gistic.txt](https://github.com/PoisonAlien/maftools/files/4308077/scores.gistic.txt)

**Comment by PoisonAlien** [comment]
Hello,
If the entry under the column `Type` is `Del` then the scores are negated for plotting purpose.

**Comment by beginner984** [comment]
Hello @PoisonAlien 

Between my two groups of patients, I have observed a higher number of CNVs in group of non-responders to chemotherapy. I have obtained total Copy number, deletion and amplification from maf object which I have created by GISTIC results (I am not sure if I have included the segmentation file too). If somebody asks me what I mean by total copy number, should I say total number of recurrently CN changed genes or total number of segments in the cohort?

Thank you so much for any thoughts
